docs/api/en/core/BufferGeometry.html

```diff
@@ -32,9 +32,21 @@ <h2>Code Example</h2>
 			-1.0,  1.0,  1.0,
 			-1.0, -1.0,  1.0
 		] );
-
+		// define normals for each vertex - in this case the square is facing us.
+		// normals are required for mesh materials to render visibly
+		const normals = new Float32Array( [
+			0, 0, 1,
+			0, 0, 1,
+			0, 0, 1,
+			
+			0, 0, 1,
+			0, 0, 1,
+			0, 0, 1,
+		] );
+			
 		// itemSize = 3 because there are 3 values (components) per vertex
 		geometry.setAttribute( 'position', new THREE.BufferAttribute( vertices, 3 ) );
+		geometry.setAttribute( 'normal', new THREE.BufferAttribute( normals, 3 ) );
 		const material = new THREE.MeshBasicMaterial( { color: 0xff0000 } );
```

Comment:
The listing should represent a minimal code example. When using the geometry like in the above setup with `MeshBasicMaterial`, the normal buffer attribute is not necessary.

Hence, I vote to close the PR. Otherwise it would only be consequent to add other attributes like `uv` and `color`, too. But then it is not a minimal code example anymore.

Reply:
Have you tested that? For me it only renders anything if I add the normals.

Reply:
https://jsfiddle.net/n8hpv4kz/

Reply:
Oh I get it now. I had accidentally mixed examples and used `MeshStandardMaterial` without noticing the difference. Sorry about that!